app/assets/javascripts/all.js

```diff
@@ -1,5 +0,0 @@
-// This is a manifest file used by the runner for JavaScript unit tests.
```

Comment:
As confirmed by @MartinJJones, this file is not used anymore, so it can be removed.

Reply:
Just some history on modernizing the JS test approach in frontend app:
- https://github.com/alphagov/frontend/pull/1414 - 2018
added to jasmine.yml file
- https://github.com/alphagov/frontend/pull/3221 - 2022
jasmine.yml file removed, app.js not moved to jamine-browser.json

Reply:
Removal of all.js has been extracted to a separate PR.